README.md

```diff
@@ -67,6 +69,19 @@ This app is a mobile web application where you can manage your budget: you have
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budgetbuddy-production.up.railway.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
This might be a network or server issue on your side. On my end, I can create the category without any difficulties. You can try reloading the page. I was also experiencing some application failed-to-respond errors but I managed to fix them but they were being caused by something totally unrelated